pyaedt/misc/misc.py

```diff
@@ -16,3 +16,50 @@ def list_installed_ansysem():
         )
 
     return version_list
+
+
+def installed_versions():
+    """Installed AEDT versions.
+    It returns a dictionary, version is the key, installation path is the value."""
+
+    return_dict = {}
+    version_list = list_installed_ansysem()
+    for version_env_var in version_list:
+        if "ANSYSEMSV_ROOT" in version_env_var:
+            current_version_id = version_env_var.replace("ANSYSEMSV_ROOT", "")
+            student = True
+        else:
+            current_version_id = version_env_var.replace("ANSYSEM_ROOT", "")
+            student = False
+        try:
+            version = int(current_version_id[0:2])
+            release = int(current_version_id[2])
+            if version < 20:
+                if release < 3:
+                    version -= 1
+                else:
+                    release -= 2
+            if student:
+                v_key = "20{0}.{1}SV".format(version, release)
+            else:
+                v_key = "20{0}.{1}".format(version, release)
+            return_dict[v_key] = os.environ[version_env_var]
+        except:  # pragma: no cover
+            pass
+    return return_dict
+
+
+def current_version():
+    """Current AEDT version."""
+    try:
+        return list(installed_versions().keys())[0]
+    except (NameError, IndexError):
+        return ""
+
+
+def current_student_version():
+    """Current student AEDT version."""
```

Comment:
```suggestion
    """Get the current AEDT student version."""
```
the words "student version" should be kept together, although I don't know if we want to capitalize those words. We are currently inconsistent.